<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="2">
    <widget class="QLabel" name="PassConLbl">
     <property name="text">
      <string>تکرار رمز عبور*</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="AddressLe"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="SexCombo">
     <item>
      <property name="text">
       <string>انتخاب</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>مرد</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>زن</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>مایل به گفتن نیستم</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="FNameLe"/>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="EmailLbl">
     <property name="text">
      <string>ایمیل*</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLineEdit" name="PassLe">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLineEdit" name="PassConLe">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="PhoneLbl">
     <property name="text">
      <string>تلفن*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="SexLbl">
     <property name="text">
      <string>جنسیت*</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="ShowPassCB">
     <property name="text">
      <string>نمایش رمز عبور</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="FNameLbl">
     <property name="text">
      <string>نام خانوادگی*</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="CancelBtn">
     <property name="text">
      <string>انصراف</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="PassLbl">
     <property name="text">
      <string>رمز عبور*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLineEdit" name="UsernameLe"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="EmailLe"/>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="OkBtn">
     <property name="text">
      <string>ثبت نام</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="NameLbl">
     <property name="text">
      <string>نام*</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="AdressLbl">
     <property name="text">
      <string>آدرس*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="UsernameLbl">
     <property name="text">
      <string>نام کاربری*</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="PhoneLe"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="NameLe">
     <property name="locale">
      <locale language="Persian" country="Iran"/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="NationalCodeLbl">
     <property name="text">
      <string>کد ملی*</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="NationalCodeLe">
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>SignUpDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
